shop/src/main/java/ro/msg/learning/shop/domain/key/OrderDetailId.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.domain.key;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import lombok.*;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+@Embeddable
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Data
```

Comment:
Again:
1. If we need `@ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor`, but ALL of them we use `@Data`
2. If we don't need all of them we specify only the necessary annotations